test.ts

```diff
@@ -1,3 +1,4 @@
+'use strict';
```

Comment:
`use strict` combined with `Object.freeze` will throw an error if you attempt to mutate an object, so adding this `use strict` to the top here enables that functionality for the rest of the tests in this file.